DDDetectors/src/Mask_o1_v01_geo.cpp

```diff
@@ -58,6 +69,9 @@ static dd4hep::Ref_t create_element(dd4hep::Detector& description,
   if (xmlParameter.hasAttr(_Unicode(rotationX)))
     rotationX = xmlParameter.attr< bool >(_Unicode(rotationX));
 
+  int counter = 0; 
+  Material sectionMat;
+  bool isSensitive = false;
```

Comment:
this variable declaration has to be inside the loop
(if you have a sensitive element followed by a passive one, isSensitive is still true at the moment)